pkg/resolver/testHelper_test.go

```diff
@@ -48,35 +49,75 @@ func (r *Row) Scan(dest ...interface{}) error {
 // https://github.com/jackc/pgx/blob/master/rows.go#L24
 // ====================================================
 
-func newMockRows(mockDataFile string) *MockRows {
-	// Read json file and build mock data
-	bytes, _ := ioutil.ReadFile("./mocks/mock.json") //read data into Items struct which is []map[string]interface{}
-	var resources map[string]interface{}
-	if err := json.Unmarshal(bytes, &resources); err != nil {
-		panic(err)
-	}
+func newMockRows(testType string) *MockRows {
+	if testType == "non-rel" {
 
-	items := resources["addResources"].([]interface{})
+		dataDir := "./mocks/mock.json"
+		bytes, _ := ioutil.ReadFile(dataDir)
+		var data map[string]interface{}
+		if err := json.Unmarshal(bytes, &data); err != nil {
+			panic(err)
+		}
+		items := data["addResources"].([]interface{})
+		mockData := make([]map[string]interface{}, len(items))
+		for i, item := range items {
+			uid := item.(map[string]interface{})["uid"]
+			mockData[i] = map[string]interface{}{
+				"uid":     uid,
+				"cluster": strings.Split(uid.(string), "/")[0],
+				"data":    item.(map[string]interface{})["properties"],
+			}
+		}
+		columnHeaders := []string{"uid string", "cluster string", "data *map[string]interface{}"}
```

Comment:
Remove the types from here because Scan() is inferring it from the arg type.  I think that's better than declaring it here.